src/__tests__/waitFor.test.tsx

```diff
@@ -204,6 +204,37 @@ test.each([false, true])(
   }
 );
 
+const fibonaci = (n: number): number => {
+  if (n === 0 || n === 1) {
+    return 1;
+  }
+
+  return fibonaci(n - 1) + fibonaci(n - 2);
+};
+
+test.each([false, true])(
+  'it should not depend on real time when using fake timers (legacyFakeTimers = %s)',
+  async () => {
+    jest.useFakeTimers({ legacyFakeTimers: false });
```

Comment:
This test does not seem to use `each` parameter. Pls remove each param or apply it to `legacyFakeTimers`

Reply:
Oh my bad, fixed it